src/libs/tracing/timelineitemsrenderpass.cpp

```diff
@@ -29,6 +29,16 @@
 #include <QSGVertexColorMaterial>
 #include <QtAlgorithms>
 
+#ifdef __OS2__
+// YD hack
```

Comment:
Please be more specific what this hack is for and why. And no need to put your initials here as they are in the commit anyway :)

Reply:
this was not intended to be committed so early...